src/plugins/index_pattern_field_editor/public/components/field_format_editor/editors/url/url.tsx

```diff
@@ -59,9 +58,6 @@ export class UrlFormatEditor extends DefaultFormatEditor<
 > {
   static contextType = contextType;
   static formatId = 'url';
```

Comment:
Small problem with URL formatter: it opens its help in the second flyout. You can't close it without closing both flyouts. This could easily lead to lost edits

Reply:
I've linked to online docs and removed the flyouts.